docs/artefacts/offramps.md

```diff
@@ -487,6 +486,62 @@ Compare And Swap operation. Those operations require old values to match what it
 }}
 ```
 
+
+### dns
```

Comment:
can you move this section so we maintain alphabetical order?
You also need to add it to the table at the top of the file.